src/server/webhookServer.ts

```diff
@@ -0,0 +1,74 @@
+import * as http from 'http';
+import _debug from 'debug';
+import { LIBRARY_NAME } from '../constants';
+import { IAlice } from '../alice';
+
+const debug = _debug(LIBRARY_NAME);
+
+export interface IWebhookServerConfig {
+  port: number;
+  webhookUrl: string;
+  options: object;
+  handleRequest: IAlice['handleRequest'];
```

Comment:
It's not a good idea to link IAlice and IWebhookServerConfig intefaces.

Better something like (request: IApiRequest): IApiResponse

Reply:
done